test/integration/026_aliases_test/macros/cast.sql

```diff
@@ -0,0 +1,13 @@
+
+
+{% macro string_literal(s) -%}
+  {{ adapter_macro('test.string_literal', s) }}
+{%- endmacro %}
+
+{% macro default__string_literal(s) %}
+    '{{ s }}'::text
+{% endmacro %}
+
+{% macro bigquery__string_literal(s) %}
+    cast('{{ s }}' as string)
+{% endmacro %}
```

Comment:
random: let's move this macro https://github.com/fishtown-analytics/dbt-utils/blob/master/macros/cross_db_utils/safe_cast.sql from dbt-utils to dbt proper.

Reply:
yeah, agree, that would be super useful. Do you think that should happen in this PR or a different one?